cmake/define-test.cmake

```diff
@@ -142,7 +142,7 @@ endfunction()
 #                  cmake test infrastructure.
 function(define_test)
     # Parse options
-    set(options NEEDS_LD_WRAP NOT_IN_CHECK_IA2 NO_LIBS CRITERION_TEST)
+    set(options NEEDS_LD_WRAP NOT_IN_CHECK_IA2 NO_LIBS CRITERION_TEST WITHOUT_RUNTIME)
```

Comment:
Please add to doc string

Reply:
Also, what do you think about `WITHOUT_SANDBOX`?

Reply:
Done in d542238.